src/main/java/frc/robot/RobotContainer.java

```diff
@@ -56,6 +59,8 @@ private void configureButtonBindings() {
     new AnalogButton(xbox, 3).whenPressed(new InstantCommand(()->m_driveTrain.setSpeed(Constants.mikhailSpeed))).whenReleased(new InstantCommand(()->m_driveTrain.setSpeed(1)));
     //binds y to output and runs when held
     new JoystickButton(xbox, Button.kY.value).whenPressed(new InstantCommand(m_output::startWheel, m_output)).whenReleased(new InstantCommand(m_output::stopWheel, m_output));
+    // Maps the Intake spinny thing to the 'A' button
```

Comment:
don't use the word "thing", look at the comment for the command above for reference.